packages/shared/src/UI/components/Image/index.tsx

```diff
@@ -41,6 +41,8 @@ export interface ImageProps
     fallback?: URL | string | JSX.Element
     disableSpinner?: boolean
     containerProps?: HTMLProps<HTMLDivElement>
+    /** If optimistic is true, means that the resource is loadable */
+    optimistic?: boolean
```

Comment:
this is not a good name. maybe more clear, for example, `preferNative`

Reply:
Like the Optimistic UI https://www.apollographql.com/docs/react/v2/performance/optimistic-ui/